app/services/participant_validation_service.rb

```diff
@@ -55,7 +56,9 @@ def full_dqt_client
   end
 
   def matching_record(trn:, nino:, full_name:, dob:)
-    return if trn.blank?
+    return if trn.blank? && nino.blank?
+
+    trn ||= "1"
```

Comment:
Can you explain why you need to default this? As far as I can see, you will never set this because you return if trn is blank.